src/components/Input/Textarea.test.tsx

```diff
@@ -0,0 +1,63 @@
+
+import { shallow } from 'enzyme';
+import * as React from 'react';
+import { Textarea } from './Textarea';
+
+describe('<Texarea />', () => {
+    describe("Rendering", () => {
+        it('should render without throwing an error', () => {
+            const cut = shallow(<Textarea label="Label" />);
+            expect(cut.find('textarea')).toHaveLength(1);
+        });
+        it('should render the label', () => {
+            const cut = shallow(<Textarea label="Label" />);
+            expect(cut.find('label').text()).toEqual("Label");
+        });
+        it('should associate the label with the input', () => {
+            const cut = shallow(<Textarea id="the_id" label="Label" />);
+            expect(cut.find('textarea').prop('id')).toEqual("the_id");
+            expect(cut.find('label').prop('htmlFor')).toEqual("the_id");
+        });
+        it('should apply custom class', () => {
+            const cut = shallow(<Textarea className="my-class" label="Label" />);
+            expect(cut.find('div.rvt-input').hasClass("my-class")).toEqual(true);
+        });
+    });
+    describe("Notes", () => {
+        it('should apply the note', () => {
+            const cut = shallow(<Textarea note="Note" label="Label" />);
+            expect(cut.find('small').text()).toEqual("Note");
+        });
+        it('should associate the input with the note', () => {
+            const cut = shallow(<Textarea id="the_id" note="Note" label="Label" />);
+            expect(cut.find('textarea').prop('aria-describedby')).toEqual("the_id_note");
```

Comment:
yeah I think you're aiming for something like this https://rivet.uits.iu.edu/components/forms/text-input/#textarea-inline-validation-states maybe?